### PR TITLE
chore: add CONSENT and THIRD_PARTY_LINK to thirdparty variant PartyId…

### PR DESCRIPTION
**Pull request by kleyow**
…Type

### CODE REVIEW

**Comment by kleyow** [comment]
https://github.com/mojaloop/project/issues/1964 Discussion still ongoing but this was an easy change to write up so I'll just leave it ready until I need it for when I connect the services and write up tests.

File: src/thirdparty/openapi.ts [hunk 1]
```diff
@@ -681,23 +718,24 @@ export interface components {
...
-      /**
-       * Identifier of the Party.
-       */
-      partyIdentifier: string;
-      /**
-       * Either a sub-identifier of a PartyIdentifier, or a sub-type of the
-       * PartyIdType, normally a PersonalIdentifierType.
-       */
-      partySubIdOrType?: string;
+      partyIdentifier: components['schemas']['PartyIdentifier'];
```
Comment (lewisdaly) [inline src/thirdparty/openapi.ts:730]: nit: please leave in the nice comments we had above these fields

**Review by lewisdaly — APPROVED** [review]

File: src/thirdparty/openapi.ts (hunk 1 above)
Comment (kleyow) [reply src/thirdparty/openapi.ts:730]: this file is generated when building, it seems it just moves the comments around https://github.com/mojaloop/api-snippets/pull/46/files/2f43eee51f649905b037eb61029d7634d056af93#diff-978850faba52906f4cc439ebc467fdc887055871fe18358001ec0c3c0af17a79R807

Comment (lewisdaly) [reply src/thirdparty/openapi.ts:730]: ok cool, good to know

**Pull request merged**